jenkins/ci.suse.de/templates/cloud-crowbar-pipeline-template.yaml

```diff
@@ -123,12 +123,6 @@
             NOTE: use this parameter only if you want to use a generated input model. To use an existing input model instead,
             leave this field empty and use the 'model' parameter instead.
 
-      - string:
-          name: scenario_file
```

Comment:
I would rather keep this as an override option. Sometimes it is easier to quickly test something with a custom scenario than modify the scenario generator. E.g. core (non-openstack) barclamps are not included in this version of generator.

Reply:
@skazi0 The custom Crowbar batch file would have to be really similar to the one that would be otherwise generated from the selected scenario (node number, node names, cluster names etc), otherwise it won't work. I will keep this as an option, but with a big warning that it's a really advanced feature.

Reply:
@skazi0 I changed my mind, I don't think it's such a good idea to maintain this feature anymore. The unified CI is already as complex as it gets and if anyone really needs to use personalized batch scenarios, then the localhost Crowbar ECP deployment scripts can be used to run deployment steps manually and edit the generated batch file as needed.

Reply:
Sure. Go ahead. This was just my opinion :o)